CONTRIBUTING.md

```diff
@@ -2,17 +2,21 @@
 
 ### Development install
 
-Note: You will need NodeJS to build the extension package.
+**Note:** You will need [Node.js](https://nodejs.org/) and [Docker](https://www.docker.com/) to build the extension package.
```

Comment:
```suggestion
**Note:** You will need [Node.js](https://nodejs.org/) to build the extension package.
```

Question: Do we need Docker as a pre-requisite for installation? As I had it pre installed, I might have missed it. Feel free to commit the suggestion in case we don't need Docker as a pre installed requirement.

Reply:
No, this is not needed as far as I know.